gems/aws-crt-auth/lib/aws-crt-auth/credentials.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Aws
+  module Crt
+    module Auth
+      # Utility class for Credentials.
+      # @api private
+      class Credentials
+        UINT64_MAX = 18_446_744_073_709_551_615
+
+        # @param [String] access_key_id
+        # @param [String] secret_access_key
+        # @param [String] session_token (nil)
+        # @param [Time|int] expiration (nil)
```

Comment:
note that expiration, when it's an int, is a timepoint in "seconds since unix epoch" or however it's phrased in Ruby Town

Reply:
Yeah - this is missing detailed documentation for all of the members...